docs: Add RequeueAfterSeconds documentation for git generator (#22890)
argoproj/argo-cd: docs/operator-manual/applicationset/Generators-Git.md

## Webhook Configuration

> When using a Git generator, ApplicationSet polls Git repositories every every `requeueAfterSeconds` interval (defaulting to every three minutes) to detect changes. To eliminate

Reviewer: Or perhaps even 

```suggestion
When using a Git generator, the ApplicationSet controller polls Git repositories every 3 minutes (this can be customized per ApplicationSet with `requeueAfterSeconds`) to detect changes. To eliminate
```

## Webhook Configuration

> When using a Git generator, the ApplicationSet controller polls Git repositories every 3 minutes (this can be customized per ApplicationSet with `requeueAfterSeconds`) to detect changes. To eliminate

Reviewer 2: ```suggestion
When using a Git generator, the ApplicationSet controller polls Git repositories every 3 minutes to detect changes. This interval can be customized for each ApplicationSet using the `requeueAfterSeconds` field. To eliminate
```
